sbt-cross/src/main/scala/sbtcross/JvmCross.scala

```diff
@@ -0,0 +1,37 @@
+package sbtcross
+
+import sbt._
+
+case object JVMPlatform extends Platform {
+  def identifier: String = "jvm"
+  def enable(project: Project): Project =
+    project.settings(crossPlatform := JVMPlatform)
+  val crossBinary: CrossVersion = CrossVersion.binary
+  val crossFull: CrossVersion   = CrossVersion.full
+}
+
+trait JvmCross {
+  val JVMPlatform = sbtcross.JVMPlatform
+
+  implicit class JvmCrossProjectBuilder(builder: CrossProject.Builder) {
+    private def build = builder.crossType(CrossType.Full)
```

Comment:
You should not have this; it will result in a lot of code duplication. Instead you should have:
```scala
implicit def CrossProjectBuilderJVMOps(builder: CrossProject.Builder): CrossProjectJVMOps =
  new JVMCrossProjectOps(builder.crossType(CrossType.Full))
```